homeassistant/components/comfoconnect/sensor.py

```diff
@@ -264,7 +304,7 @@ def __init__(self, name, ccb: ComfoConnectBridge, sensor_type) -> None:
         """Initialize the ComfoConnect sensor."""
```

Comment:
Good idea! I'll change that. Is it also recommended to use `_attr_should_poll = False` instead of the following?

```py
    @property
    def should_poll(self) -> bool:
        """Do not poll."""
        return False
```

Reply:
I did this anyway, it seems like a good idea.